python/ray/worker.py

```diff
@@ -764,7 +764,6 @@ def init(
         log_to_driver=log_to_driver,
         worker=global_worker,
         driver_object_store_memory=_driver_object_store_memory,
-        job_id=None,
```

Comment:
should this be changed?

Reply:
This should not be changed. I will fix it.